src/reducers/search.js

```diff
@@ -38,7 +38,7 @@ const entries = (state = defaultState, action) => {
         map.set('isFetching', false);
         map.set('page', page);
         map.set('term', searchTerm);
-        map.set('entryIds', page === 0 ? entryIds : map.get('entryIds', List()).concat(entryIds));
+        map.set('entryIds', (!page || page === 0) ? entryIds : map.get('entryIds', List()).concat(entryIds));
```

Comment:
When we're using cursor-based pagination, we do *not* want the `entryIds` to be appended to, since we only model knowledge of one set of entries and one cursor, which we retrieve from the backend.